packages/sui-analytics/src/anonymousUser/index.js

```diff
@@ -0,0 +1,22 @@
+import cookieStrategy from './cookies'
+import localStorageStrategy from './localStorage'
+import uuid from 'uuid/v4'
+
+const KEY = 'sui-analytics-anonymous-user-id'
+
+const getAnonymousUserId = function(key) {
+  return (
+    cookieStrategy.getItem(key) || localStorageStrategy.getItem(key) || uuid()
```

Comment:
Maybe a try/catch strategy here could make sense. There are scenarios like Safari InPrivate mode where try to access to LS will throw an exception.